Python/_warnings.c

```diff
@@ -1149,8 +1158,16 @@ warnings_warn_explicit_impl(PyObject *module, PyObject *message,
             return NULL;
         }
     }
+
+    WarningsState *st = warnings_get_state(tstate->interp);
+    if (st == NULL) {
+        return NULL;
+    }
```

Comment:
These `st == NULL` checks don't do anything because `warnings_get_state()` can't return NULL. It returns a pointer to an embedded struct.

Reply:
I converted the ones I touched to asserts.